_data/entertainment.yml

```diff
@@ -30,30 +32,35 @@ websites:
     - name: hitbox
       url: https://www.hitbox.tv/
       twitter: hitboxliveHelp
+      facebook: hitboxlive
       img: hitbox.png
       tfa: No
 
     - name: Hulu
       url: http://www.hulu.com/
       twitter: hulu_support
+      facebook: hulu
       img: hulu.png
       tfa: No
 
     - name: IMDb
       url: https://www.imdb.com
       img: imdb.png
       twitter: IMDb
+      facebook: imdb
       tfa: No
 
     - name: Last.fm
       url: http://www.last.fm/
       twitter: lastfmsupport
+      facebook: lastfm
       img: lastfm.png
       tfa: No
 
     - name: Netflix
       url: https://www.netflix.com/us/
       twitter: Netflixhelps
+      facebook: netflixus
```

Comment:
I think [netflix](https://www.facebook.com/netflix) should be used instead as not everyone whom visits twofactorauth.org will be from the US. Thoughts?